frontend/src/components/MentorForm.tsx

```diff
@@ -0,0 +1,79 @@
+import { useState, ChangeEvent, FC } from 'react';
+import FormContainer from './FormContainer';
+import FormRow from './FormRow';
+import { Flex, InputGroup, Button } from '@chakra-ui/react';
+import { useNavigate } from 'react-router-dom';
+
+interface Field {
+    inputType: string;
+    label: string;
+    name: string;
+}
+
+interface MentorFields {
+    mentorFields: Field[] 
+}
```

Comment:
We've got some duplication happening here. Export fields into a shared file and then make MentoreFields/MenteeFields a type which = Field (rather than an interface as we don't intend on extending them)

Aside from that Mentor/MenteeInfo can be created into a base interface which then extends the required differing fields. That way if we need to add more to the form we only need to do it in one place